Sprint-1/explore/chrome.md

```diff
@@ -11,8 +11,11 @@ In the Chrome console,
 invoke the function `alert` with an input string of `"Hello world!"`;
 
 What effect does calling the `alert` function have?
+/_ The alert function creates a pop-up window notification _/
 
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
 What effect does calling the `prompt` function have?
+/_ Prompt creates an input field for the user to enter information that should be stored as a variable _/
 What is the return value of `prompt`?
+/_ When returned, the prompted information value could be called by its assigned variable name _/
```

Comment:
How can you tell if the user clicked "OK" or "Cancel"?
 
Also, your description "When returned, the prompted information value could be called by its assigned variable name" does not quite make sense because you were using some terminology not commonly used in programming. You could also just say "The value returned by `prompt()` is ....". Alternatively, you can ask ChatGPT to see how else things can be described.